demo/timestamp/timestamp.php

```diff
@@ -0,0 +1,25 @@
+<?php
+
+require_once __DIR__ . '/../bootstrap.php';
+
+$loop = new \React\EventLoop\StreamSelectLoop();
+
+$scheduler = new \Rx\Scheduler\EventLoopScheduler($loop);
+
+$source = \Rx\Observable::interval(1000, $scheduler)
+    ->timestamp()
+    ->map(function ($x) {
+        return $x->value . ':' . $x->timestamp;
+    })
+    ->take(5);
+
+$source->subscribe($createStdoutObserver());
+
+$loop->run();
+
+// Next value: 0:1460781738354
```

Comment:
The expected output will be different every time, that is why there is no expect file.